doc/content/the-things-stack/concepts/networking/_index.md

```diff
@@ -13,40 +13,41 @@ aliases: [/reference/networking]
-| Application data, events | MQTT | API key, token | 1883 | 8883 |
-| Management, data, events | gRPC | API key, token | 1884 | 8884 |
-| Management | HTTP | API key, token | 1885 | 8885 |
-| Backend Interfaces | HTTP | Custom | 1886 | 8886 |
-| {{% lbs %}} LNS | Web Sockets | Auth Token, Custom | 1887 | 8887 |
-| Tabs Hubs LNS {{< distributions "Cloud" "Enterprise" >}} | Web Sockets | Auth Token, Custom | 1888 | 8888 |
+| **Purpose**                                              | **Protocol**                                                                                      | **Authentication** | **Port**   | **Port (TLS)** |
+| -------------------------------------------------------- | ------------------------------------------------------------------------------------------------- | ------------------ | ---------- | -------------- |
+| Gateway data                                             | [Semtech Packet Forwarder](https://github.com/Lora-net/packet_forwarder/blob/master/PROTOCOL.TXT) | None               | 1700 (UDP) | N/A            |
+| Gateway data                                             | MQTT (V2)                                                                                         | API key, token     | 1881       | 8881           |
+| Gateway data                                             | MQTT                                                                                              | API key, token     | 1882       | 8882           |
+| Application data, events                                 | MQTT                                                                                              | API key, token     | 1883       | 8883           |
+| Management, data, events                                 | gRPC                                                                                              | API key, token     | 1884       | 8884           |
+| Management                                               | HTTP                                                                                              | API key, token     | 1885       | 8885           |
+| Backend Interfaces                                       | HTTP                                                                                              | Custom             | 1886       | 8886           |
+| {{% lbs %}} LNS                                          | Web Sockets                                                                                       | Auth Token, Custom | 1887       | 8887           |
+| Tabs Hubs LNS {{< distributions "Cloud" "Enterprise" >}} | Web Sockets                                                                                       | Auth Token, Custom | 1888       | 8888           |
+| CUPS mTLS {{< distributions "Cloud" "Enterprise" >}}     | HTTP                                                                                              | mTLS               | NA         | 8987           |
```

Comment:
Which part is not open source?

Reply:
I'm not sure I understand the question. This indicates that the port `8987` is not available in OS.